### DISCUSSION TITLE
Welcome to fedcloudclient Discussions!

### DISCUSSION THREAD

**Discussion by tdviet** [tdviet/fedcloudclient#93] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sebastian-luna-valero** [comment]
Hi!

I am Sebastian, and I work as Cloud Community Support Specialist at [EGI Foundation](https://www.egi.eu).

`fedcloudclient` is a great piece of software and I use it to check information and perform tests across the EGI Federated Cloud.

Best regards,
Sebastian
